### PR TITLE
Readme suggestions

### PR DESCRIPTION
**Pull request by LawrenceLoz**
A few suggested changes here:

- Change to the openssl parameter syntax, looks like a typo? (I generated using nodes and cert and key behave as expected)
- Change to suggested callback URL for Heroku to remove port specification. Using the suggested version with port specified I wasn't able to get Heroku to recognise the callback. Reading a bit (e.g. https://stackoverflow.com/questions/28706180/setting-the-port-for-node-js-server-on-heroku) I think this is because Heroku expects to always use default ports, and even with the https.createServer() function it doesn't seem to be listening at port 8081 - using original callback URL I can see the network call to the callback URL but no indication Heroku has recognised this. Using a callback URL of https://my-heroku-app.herokuapp.com:443/oauthcallback works, as does https://my-heroku-app.herokuapp.com/oauthcallback
- Recommendation to enable device flow in connected app

### CODE REVIEW

**Comment by LawrenceLoz** [comment]
Not sure whether I'm missing something around Heroku config or anything else to get the listener at port 8081 to work - curious if this works for you?

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Be aware that this isn't safe.
 
 -   Install openssl by following the [instructions](https://github.com/openssl/openssl#build-and-install) on its Github repository.
 -   Clone this repository to your local machine.
--   Run the following command in the root of the cloned repository: `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out server.crt -days 365 -noenc`
+-   Run the following command in the root of the cloned repository: `openssl req -x509 -newkey rsa:2048 -keyout key.pem -out server.crt -days 365 -nodes`
```
Comment (NVandenBossche) [inline README.md:37]: The -nodes flag is deprecated since OpenSSL 3.0. See https://www.openssl.org/docs/manmaster/man1/openssl-req.html

**Review by NVandenBossche — CHANGES_REQUESTED** [review]
Thanks, indeed the port was incorrect :-) Still some changes I need to do to the readme + add some audio to the instruction video.

The only thing I would change (back) is the nodes/noenc (nodes is deprecated since 3.0)

File: README.md (hunk 1 above)
Comment (LawrenceLoz) [reply README.md:37]: Ah yes you're right, although openssl 3.0 isn't yet offered as a stable version (https://www.openssl.org/source/). Using v1.1.1 I get an "Unrecognized flag" error with -noenc... I could change to mention both versions with a description of why each one might be needed?

**Comment by NVandenBossche** [comment]
Closing because I made the changes manually myself

**Pull request closed without merging**